javascript/javascript1/week2/flight.js

```diff
@@ -0,0 +1,8 @@
+//Flight booking fullname function
+let firstName= prompt("What is your first name?");
+let lastName= prompt("What is your last name?");
+function getFullname(firstName,lastName){
+    return firstName + " "+ lastName;
```

Comment:
Great job!
Only a suggestion though, Template literals allow you to embed expressions inside a string by wrapping the expressions in ${}.
Try this template, it is useful when using JS
```suggestion
    return `${firstName} ${lastName}`;
```
Why Use Template Literals?
Cleaner and more intuitive than concatenation (+).
Simplifies working with dynamic content in strings.
Helps avoid errors when constructing complex strings.

Reply:
Understandable. Thanks :)